src/store/FlowerStore.java

```diff
@@ -0,0 +1,77 @@
+package store;
+
+import java.util.ArrayList;
+
+public class FlowerStore {
+    private ArrayList<String> firstBouquet;
+    private ArrayList<String> secondBouquet;
+    private int walletAfter;
+    private int walletBefore = 1000;
+
+    public ArrayList<String> sell(int countOfRose, int countOfCamomile, int countOfTulip) {
+        int i = countOfRose + countOfCamomile + countOfTulip;
+        ArrayList<String> firstListofFlowers = new ArrayList<>();
+        int j = 0;
+        while (j < countOfRose) {
```

Comment:
better condition (countOfRose != 0) because you do nothing with variable "j" and it is embarrassed